### ISSUE TITLE
Check if all characters are ascii

### ISSUE THREAD

**Issue by joachimwolff** [deeptools/HiCExplorer#175]
As reported in #174 our software crashes if non-ASCII characters are used. We should add a check for this and maybe remove non-ascii characters or if possible replace them to the closest ASCII character, e.g. the german `ü` with `u`.

**Comment by joachimwolff** [comment]
Implemented with release 2.1.

**Comment by hwick** [comment]
Hello,
I am getting a unicode error similar to the one that appears in issue #174 when trying to run hicBuildMatrix. None of my file names have non-ASCII characters as far as I am aware


```
hicBuildMatrix --samFiles /path/to/Ctrl_R1_hg19.bwt2merged.bam /path/to/Ctrl_R2_hg19.bwt2merged.bam \
                 --binSize 10000 \
                 --restrictionSequence AAGCTT \
                 --threads 4 \
                 --inputBufferSize 100000 \
                 --outBam /path/to/Ctrl.bam \
                 -o /path/to/Ctrl_matrix.h5 \
                 --QCfolder /path/to/hicQC
```
error:
```
Traceback (most recent call last):
  File "/cm/shared/jhmi/apps/HiCExplorer/python2.7.6/64/2.0/bin/hicBuildMatrix", line 7, in <module>
    main()
  File "/cm/shared/jhmi/apps/python/gcc/64/2.7.6/lib/python2.7/site-packages/hicexplorer/hicBuildMatrix.py", line 1465, in main
    QC.main("-l {} -o {}".format(log_file_name, args.QCfolder).split())
  File "/cm/shared/jhmi/apps/python/gcc/64/2.7.6/lib/python2.7/site-packages/hicexplorer/hicPrepareQCreport.py", line 284, in main
    read_orientation_table, table)
  File "/cm/shared/jhmi/apps/python/gcc/64/2.7.6/lib/python2.7/site-packages/hicexplorer/hicPrepareQCreport.py", line 65, in save_html
    .format(lambda x: '{:,}'.format(x) if x > 1 else '{:.2%}'.format(x)).render())
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 1241: ordinal not in range(128)
```
